src/software/pipeline/main_tracksBuilding.cpp

```diff
@@ -136,6 +136,25 @@ int aliceVision_main(int argc, char** argv)
     track::TracksMap mapTracks;
     tracksBuilder.exportToSTL(mapTracks);
 
+    //Fill additional data
+    for (auto & ptrack : mapTracks)
+    {
+        auto & track = ptrack.second;
+
+        for (auto & pitem : track.featPerView)
+        {
+            IndexT viewId = pitem.first;
+            track::TrackItem & item = pitem.second;
+            const auto & feats = featuresPerView.getFeaturesPerDesc(viewId);
+
+            const feature::PointFeatures & features = feats.at(track.descType);
+            const feature::PointFeature & feature = features.at(item.featureId);
+
+            item.coords = feature.coords().cast<double>();
+            item.scale = feature.scale();
+        }
+    }
+
```

Comment:
Could be a function in track, instead of being implemented in the main?
Is it not possible to fill it directly in the "TracksBuilder::build"?

Reply:
since we are in C++17 we can use structured binding and write
```c++
        for (auto&& [viewId, TrackItem] : track.featPerView)
        {
            const auto & feats = featuresPerView.getFeaturesPerDesc(viewId);

            const feature::PointFeatures & features = feats.at(track.descType);
            const feature::PointFeature & feature = features.at(item.featureId);

            item.coords = feature.coords().cast<double>();
            item.scale = feature.scale();
        }
    }

```